@xen-orchestra/web/src/components/vm/dashboard/VmDashboardCpuUsageChart.vue

```diff
@@ -73,5 +73,11 @@ const maxValue = computed(() => {
   return Math.ceil(maxCpuUsage / 100) * 100
 })
 
-const valueFormatter: ValueFormatter = value => n(value / 100, 'percent')
+const valueFormatter: ValueFormatter = value => {
+  if (value === null) {
+    return ''
+  }
+
+  return n(value / 100, 'percent')
```

Comment:
Maybe limit the length to 2 digits.
```suggestion
  return n(Math.round(value / 100) * 100, 'percent')
```

Reply:
It does weird things when I put this on.
![image](https://github.com/user-attachments/assets/f2f8ab56-095e-4be2-90a5-5e494a203ef1)